Makefile

```
@@ -1,4 +1,33 @@
-.PHONY: grpc grpc-hot lint test test-verbose test-coverage test-coverage-html test-clean security
+TEST_DIR := ./internal/test
+
+.PHONY:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)